### ISSUE TITLE
F tensorflow/python/lib/core/py_exception_registry.cc:28] Check failed: singleton_ == nullptr PyExceptionRegistry::Init() already called

### ISSUE THREAD

**Issue by Vurtnel** [Gourieff/ComfyUI-ReActor#153]
### First, confirm

- [x] I have read the [instruction](https://github.com/Gourieff/ComfyUI-ReActor/blob/main/README.md) carefully
- [x] I have searched the existing issues
- [x] I have updated the extension to the latest version

### What happened?

when i startup comfyui and import comfyui-reactor， startup error, error msg is:

Prestartup Skipping comfyui-faceaging due to disable-custom-nodes

Prestartup times for custom nodes:
   0.0 seconds: /ComfyUI/custom_nodes/ComfyUI-Easy-Use

Checkpoint files will always be loaded safely.
Total VRAM 22503 MB, total RAM 59443 MB
pytorch version: 2.5.1+cu121
xformers version: 0.0.29.post1
Set vram state to: LOW_VRAM
Device: cuda:0 NVIDIA A10 : cudaMallocAsync
Using xformers attention
Python version: 3.10.16 (main, Dec 11 2024, 16:24:50) [GCC 11.2.0]
ComfyUI version: 0.3.43
ComfyUI frontend version: 1.23.4
[Prompt Server] web root: /opt/conda/lib/python3.10/site-packages/comfyui_frontend_package/static
Total VRAM 22503 MB, total RAM 59443 MB
pytorch version: 2.5.1+cu121
xformers version: 0.0.29.post1
Set vram state to: LOW_VRAM
Device: cuda:0 NVIDIA A10 : cudaMallocAsync
/opt/conda/lib/python3.10/site-packages/albumentations/__init__.py:28: UserWarning: A new version of Albumentations is available: '2.0.8' (you have '2.0.6'). Upgrade using: pip install -U albumentations. To disable automatic update checks, set the environment variable NO_ALBUMENTATIONS_UPDATE to 1.
  check_for_updates()
Nvidia APEX normalization not installed, using PyTorch LayerNorm

[ReActor] - STATUS - Running v0.6.0-a1 in ComfyUI
Torch version: 2.5.1+cu121
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
E0000 00:00:1752126717.244911    6476 cuda_dnn.cc:8310] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
E0000 00:00:1752126717.249177    6476 cuda_blas.cc:1418] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2025-07-10 05:51:59.290082: F tensorflow/python/lib/core/py_exception_registry.cc:28] Check failed: singleton_ == nullptr PyExceptionRegistry::Init() already called
Aborted (core dumped)

### Steps to reproduce the problem

Your workflow

### Sysinfo

Total VRAM 22503 MB, total RAM 59443 MB
pytorch version: 2.5.1+cu121
xformers version: 0.0.29.post1
Set vram state to: LOW_VRAM
Device: cuda:0 NVIDIA A10 : cudaMallocAsync
Python version: 3.10.16 (main, Dec 11 2024, 16:24:50) [GCC 11.2.0]
ComfyUI version: 0.3.43
ComfyUI frontend version: 1.23.4

### Relevant console log

```Shell
Prestartup Skipping comfyui-faceaging due to disable-custom-nodes

Prestartup times for custom nodes:
   0.0 seconds: /ComfyUI/custom_nodes/ComfyUI-Easy-Use

Checkpoint files will always be loaded safely.
Total VRAM 22503 MB, total RAM 59443 MB
pytorch version: 2.5.1+cu121
xformers version: 0.0.29.post1
Set vram state to: LOW_VRAM
Device: cuda:0 NVIDIA A10 : cudaMallocAsync
Using xformers attention
Python version: 3.10.16 (main, Dec 11 2024, 16:24:50) [GCC 11.2.0]
ComfyUI version: 0.3.43
ComfyUI frontend version: 1.23.4
[Prompt Server] web root: /opt/conda/lib/python3.10/site-packages/comfyui_frontend_package/static
Total VRAM 22503 MB, total RAM 59443 MB
pytorch version: 2.5.1+cu121
xformers version: 0.0.29.post1
Set vram state to: LOW_VRAM
Device: cuda:0 NVIDIA A10 : cudaMallocAsync
/opt/conda/lib/python3.10/site-packages/albumentations/__init__.py:28: UserWarning: A new version of Albumentations is available: '2.0.8' (you have '2.0.6'). Upgrade using: pip install -U albumentations. To disable automatic update checks, set the environment variable NO_ALBUMENTATIONS_UPDATE to 1.
  check_for_updates()
Nvidia APEX normalization not installed, using PyTorch LayerNorm

[ReActor] - STATUS - Running v0.6.0-a1 in ComfyUI
Torch version: 2.5.1+cu121
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
E0000 00:00:1752126717.244911    6476 cuda_dnn.cc:8310] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
E0000 00:00:1752126717.249177    6476 cuda_blas.cc:1418] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2025-07-10 05:51:59.290082: F tensorflow/python/lib/core/py_exception_registry.cc:28] Check failed: singleton_ == nullptr PyExceptionRegistry::Init() already called
Aborted (core dumped)
```
